docs/streams/developer-guide/config-streams.html

```diff
@@ -292,7 +294,7 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
               set by the user or all serdes must be passed in explicitly (see also default.value.serde).</td>
             <td><code class="docutils literal"><span class="pre">null</span></code></td>
           </tr>
-          <tr class="row-odd"><td>default.production.exception.handler</td>
+          <tr class="row-odd"><td>default.production.exception.handler (Deprecated. Use production.exception.handler instead.)</td>
```

Comment:
Wen we insert a new row here, we need update each other row below with regard to `class="row-even"` vs ` class="row-odd"` so they alternate correctly (it's annoying, but otherwise the table wont' render as intended).